### PR TITLE
Rename some confusingly named type aliases

### PR DESCRIPTION
**Pull request by AlexWaygood**
Unblocks https://github.com/PyCQA/flake8-pyi/pull/235

### CODE REVIEW

File: stdlib/_codecs.pyi [hunk 1]
```diff
@@ -8,7 +8,7 @@ from typing_extensions import Literal, TypeAlias
 class _EncodingMap:
     def size(self) -> int: ...
 
-_MapT: TypeAlias = dict[int, int] | _EncodingMap
+_Map: TypeAlias = dict[int, int] | _EncodingMap
```
Comment (JelleZijlstra) [inline stdlib/_codecs.pyi:11]: Maybe `_CharMap` would be clearer, given the `charmap_build()` function.

File: stubs/redis/redis/typing.pyi [hunk 2]
```diff
@@ -14,7 +14,7 @@ AbsExpiryT: TypeAlias = int | datetime
 ExpiryT: TypeAlias = float | timedelta
 ZScoreBoundT: TypeAlias = float | str
 BitfieldOffsetT: TypeAlias = int | str
-_StringLikeT: TypeAlias = bytes | str | memoryview
+_StringLikeT: TypeAlias = bytes | str | memoryview  # noqa: Y043
```
Comment (JelleZijlstra) [inline stubs/redis/redis/typing.pyi:17]: Add a comment explaining why we ignore the error

Comment (AlexWaygood) [reply stubs/redis/redis/typing.pyi:17]: There's already one on line 9, but I'm happy to add another here if you like?

Comment (AlexWaygood) [reply stubs/redis/redis/typing.pyi:17]: Coolio

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**